packages/block-editor/src/components/tabbed-sidebar/index.js

```diff
@@ -16,54 +21,65 @@ import { unlock } from '../../lock-unlock';
 const { Tabs } = unlock( componentsPrivateApis );
 
 function TabbedSidebar(
-	{ defaultTabId, onClose, onSelect, selectedTab, tabs, closeButtonLabel },
+	{
+		defaultTabId,
+		onClose,
+		onSelect,
+		selectedTab,
+		tabs,
+		closeButtonLabel,
+		tabsContextValue,
+		className,
+	},
 	ref
 ) {
 	return (
-		<div className="block-editor-tabbed-sidebar">
-			<Tabs
-				selectOnMove={ false }
-				defaultTabId={ defaultTabId }
-				onSelect={ onSelect }
-				selectedTabId={ selectedTab }
-			>
-				<div className="block-editor-tabbed-sidebar__tablist-and-close-button">
-					<Button
-						className="block-editor-tabbed-sidebar__close-button"
-						icon={ closeSmall }
-						label={ closeButtonLabel }
-						onClick={ () => onClose() }
-						size="small"
-					/>
+		<Tabs.Context.Provider value={ tabsContextValue }>
```

Comment:
We probably shouldn't output this unless we have a value. Maybe we don't actually need it at all?